### PR TITLE
Create spark

### PR DESCRIPTION
**Pull request by greggodwin**
I'd like to add Spark to the app directory. It doesn't have as robust a URL scheme as some other apps, but it does have one.

### CODE REVIEW

File: data/spark [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
```
Comment (pietropizzi) [inline data/spark]: This file is missing the .json ending. That’s why it isn’t showing up in the preview deploy.

Comment (greggodwin) [reply data/spark]: My bad! I'm new to GitHub and missed that part of the file name. I think I've corrected that now, but let me know if it's still not showing up. Thanks!

Comment (greggodwin) [reply data/spark]: I also grabbed the app icon from the App Store to upload it here, but I couldn't find where to do that. When I go to the directory you mentioned on the How to Contribute page and click the Upload File button, I get a message telling me that I don't have permission to upload files there. Have I missed something?

File: data/spark.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
+{
```
Comment (pietropizzi) [inline data/spark.json]: Actually: This file needs to live in `data/apps` not in just `data`. Was wondering why it wasn't showing up.

Comment (greggodwin) [reply data/spark.json]: Ah. Again, my bad. I've fixed that.

Comment (pietropizzi) [reply data/spark.json]: Cool it's showing up now: https://deploy-preview-22--app-talk.netlify.com/#spark Now only the image and then we're done here 👍

Comment (greggodwin) [reply data/spark.json]: I think I have the image in the right place now. Thanks for the help!

Comment (pietropizzi) [reply data/spark.json]: Seems like you uploaded it to your 'master' branch but this Pull Request is based on your `patch-1` branch.

**Pull request merged**